pymultinest_demo_longname.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, unicode_literals, print_function
```

Comment:
I think this script can be shortened substantially

```
from pymultinest.solve import solve

solve(lambda cube: -0.5 *(cube**2).sum(), lambda cube: cube, 2, 
	resume = True, verbose = True,
        outputfiles_basename = "a_really_really_really_really_really_really_really_really_really_really_really_really_really_really_really_really_really_long_name-")
```

Please move this file into a folder tests/